.travis.yml

```diff
@@ -30,8 +29,6 @@ matrix:
       env: SYMFONY_VERSION=3.1
     - php: 5.6
       env: SYMFONY_VERSION=3.2
-    - php: 5.3
```

Comment:
ah you looked at the php 5.4 build .. instead of the prefer-lowest .. so I guess all is well.